Lib9c.Models/States/CombinationSlotState.cs

```diff
@@ -63,22 +74,35 @@ public CombinationSlotState(IValue bencoded)
             );
         }
 
-        UnlockBlockIndex = d["unlockBlockIndex"].ToLong();
-        UnlockStage = d["unlockStage"].ToInteger();
+        UnlockBlockIndex = d[UnlockBlockIndexKey].ToLong();
 
-        if (d.TryGetValue((Text)"result", out var result))
+        if (d.TryGetValue((Text)IndexKey, out var index))
+        {
+            Index = (Integer)index;
+        }
+
+        if (d.TryGetValue((Text)ResultKey, out var result))
         {
             Result = AttachmentActionResultFactory.Create(result);
         }
 
-        if (d.TryGetValue((Text)"startBlockIndex", out var startIndex))
+        if (d.TryGetValue((Text)StartBlockIndexKey, out var value))
         {
-            StartBlockIndex = startIndex.ToLong();
+            StartBlockIndex = value.ToLong();
         }
 
-        if (d.TryGetValue((Text)"petId", out var petId))
+        if (d.TryGetValue((Text)PetIdKey, out var petId))
         {
             PetId = petId.ToNullableInteger();
         }
+
+        if (d.TryGetValue((Text)IsUnlockedKey, out var isUnlocked))
+        {
+            IsUnlocked = isUnlocked.ToBoolean();
+        }
+        else
+        {
+            IsUnlocked = Index < Nekoyume.Model.State.AvatarState.DefaultCombinationSlotCount;
```

Comment:
According to these lines, shouldn't it be just `false` (`bool`'s default value)?

https://github.com/planetarium/lib9c/blob/4c1db15000062c2f0cf1b89b659795f1181df2fe/Lib9c/Model/State/CombinationSlotState.cs#L91-L94

Reply:
I wrote this to apply the content from the link below.
https://github.com/planetarium/lib9c/blob/4c1db15000062c2f0cf1b89b659795f1181df2fe/Lib9c/Model/State/CombinationSlotState.cs#L34

Reply:
Umm, then current implementation is right. Thanks!